wee_slack.py

```diff
@@ -3604,14 +3605,25 @@ def whois_command_cb(data, current_buffer, command):
     team = EVENTROUTER.weechat_controller.buffers[current_buffer].team
     u = team.users.get(team.get_username_map().get(user))
     if u:
+        def print_profile(field):
+            value = u.profile.get(field)
+            if value:
+                team.buffer_prnt("[{}]: {}: {}".format(user, field, value))
+
         team.buffer_prnt("[{}]: {}".format(user, u.real_name))
         status_emoji = u.profile.get("status_emoji", "")
         status_text = u.profile.get("status_text", "")
         if status_emoji or status_text:
             team.buffer_prnt("[{}]: {} {}".format(user, status_emoji, status_text))
-        team.buffer_prnt("[{}]: Title: {}".format(user, u.profile.get('title', '')))
-        team.buffer_prnt("[{}]: Email: {}".format(user, u.profile.get('email', '')))
-        team.buffer_prnt("[{}]: Phone: {}".format(user, u.profile.get('phone', '')))
+
+        team.buffer_prnt("[{}]: username: {}".format(user, u.username))
+        team.buffer_prnt("[{}]: id: {}".format(user, u.identifier))
+
+        print_profile('title')
+        print_profile('email')
+        print_profile('phone')
+        print_profile('skype')
+        print_profile('fields')
```

Comment:
What is fields?

Reply:
I couldn't remember what this was, so I had to go back through the Slack API docs...

From [users.profile.get](https://api.slack.com/methods/users.profile.get):

> For a description of the `fields` key, see the users.profile.set method.

...

[users.profile.set](https://api.slack.com/methods/users.profile.set) has a description of `fields`, but it's quite lengthy so I won't paste it here.

Looking again though, I'm not even sure we get the custom profile fields in the first place.

Reply:
We get the profile with `rtm.start`, so that's why we never have to call `users.profile.get`.

Okay, after reading the docs, I see that this is custom fields each team can add. By default it only includes skype (which is the same as the skype key directly under profile). The fields are defined with an id and a value, e.g. if i add "some skype handle" as skype in my profile, fields for my profile will be this:
```
        "fields": {
          "Xf0D7H375J": {
            "value": "some skype handle",
            "alt": ""
          }
        },
```

To get what the id represent, we have to call [team.profile.get](https://api.slack.com/methods/team.profile.get).

If we want to show these fields in whois, we should do it properly, i.e. look up the ids and print them nicely. Otherwise, we should omit fields.

Reply:
I concur. I'll hide the fields for now and file a followup issue.